src/linear_algebra/bilinear_form.lean

```diff
@@ -1,26 +1,31 @@
 /-
 Copyright (c) 2018 Andreas Swerdlow. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Author: Andreas Swerdlow
+Author: Andreas Swerdlow, Kexing Ying
 -/
 
 import linear_algebra.matrix
 import linear_algebra.tensor_product
 import linear_algebra.nonsingular_inverse
+import algebra.invertible
```

Comment:
I think this import is now no longer necessary (it can move to `quadratic_form`).